src/components/Chat/ChatMessageComponent.tsx

```diff
@@ -10,8 +11,11 @@ const ChatMessageComponent = (props: IProps) => {
   const { message, author, uniqueID, files } = props.message;
   // Placeholder for when we get users
   // TODO: change when we have users, to use the username instead
-  const authorType = author === 'frivillig' ? 'self' : 'other';
-
+  const authorType = author === props.message.author ? 'self' : 'other';
```

Comment:
```suggestion
  const authorType = author === props.message.author ? 'Deg' : 'Elev';
```

Greit å ha det på Norsk 👍